docs/Data_Portal/Users_Guide/Projects.md

```diff
@@ -42,15 +42,15 @@ Clicking the link for each project name on the table will bring users to that sp
 
 [![Project Entity](images/ProjectEntity.png)](images/ProjectEntity.png "Click to see the full image.")
 
-Four buttons on the left of the header allow the user to perform a variety of actions related to the project: 
+Four buttons on the left of the header allow the user to perform a variety of actions related to the project:
 
 * __Save New Cohort__: Creates a new cohort consisting of all the cases in the project
 * __Biospecimen__: Downloads biospecimen metadata associated with all cases in the project in either TSV or JSON format
 * __Clinical__: Downloads clinical metadata about all cases in the project in either TSV or JSON format
-* __Manifest__: Downloads a manifest for all data files available in the project. The manifest can be used with the GDC Data Transfer Tool to download the files.
+* __Manifest__: Downloads a manifest for all data files available in the project. The manifest can be used with the GDC Data Transfer Tool to download the files
```

Comment:
maybe I got the rule wrong for when to put periods. I thought if the bullet contains >1 sentences, we add periods to all of them? Or do we just add to all except the last sentence?

Reply:
I think I just robotically eliminated them.  The > 1 sentence rule sounds reasonable.